### ISSUE TITLE
Insert skip to content link at very top

### ISSUE THREAD

**Issue by rzach** [vlmantova/bookml#8]
Insert a "Skip to Content" link at the top of each page on the website. This will allow screen reader users to skip the top menu directly to the beginning of the chapter content. You can see an example on this page: https://opentextbc.ca/studentsuccess/chapter/learning-action-plan/ If you start tabbing, a "Skip to content" link pops up that takes you directly to the beginning of the chapter. Here is more information: [WCAG - Bypass Blocks](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

(Note that to be effective this should be the very first link on the page; not sure how this can work with the sidebar when it is collapsed)
